solara/autorouting.py

```diff
@@ -438,7 +446,7 @@ def generate_routes(module: ModuleType) -> List[solara.Route]:
     return routes
 
 
-def generate_routes_directory(path: Path) -> List[solara.Route]:
+def generate_routes_directory(path: Path, markdown_renderer=None) -> List[solara.Route]:
```

Comment:
```suggestion
def generate_routes_directory(path: Path, markdown_component=None) -> List[solara.Route]:
```
I think makes it more clear what the argument should be (a component)